backend/pong/consumers.py

```diff
@@ -47,16 +45,18 @@ async def connect(self):
             return
 
     async def disconnect(self, close_code):
-        pass
+        await sync_to_async(self.match.abort_match)()
+        await self.close()
 
     async def receive(self, text_data):
         key_press = text_data.strip()
         self.game.update_positions(key_press)
 
         if self.game.game_stats.game_over == True:
             await self.send_positions()
-            await self.save_match_results()
+            await self.save_match_final_results()
```

Comment:
So this is supposed to be back to final result?

Reply:
There's some git shenanigans going on.